tests/unit/tf/core/test_combinators.py

```diff
@@ -103,6 +103,54 @@ def test_parallel_block_select_from_names():
         blocks.select_by_names(["0", "2"])
 
 
+def test_parallel_block_select_by_tags(music_streaming_data):
+    continuous_block = mm.Filter(Tags.CONTINUOUS)
+    embedding_block = mm.EmbeddingFeatures.from_schema(
```

Comment:
I think we could use here `Embeddings()` instead of `EmbeddingFeatures`, as the latter is going to be deprecated in the future.